fplstats/analyzers.py

```diff
@@ -749,7 +749,7 @@ def get_gw1_picks_standings(self: T, print_result=True):
             # auto-sub totals
             captain_pick = next((x for x in gw1_starting_picks if x.is_captain))
             total_captain_points = 0
-            vc_pick = next((x for x in gw1_starting_picks if x.is_vice_captain))
+            vc_pick = next((x for x in gw1_starting_picks if x.is_vice_captain), None)
```

Comment:
Tror dette kan løses ved å bytte gw1_starting_picks med gw1_all_picks. Da tror jeg ikke man trenger en fallback
```suggestion
            vc_pick = next((x for x in gw1_all_picks if x.is_vice_captain))
```

Reply:
Bekrefter at det funker så har oppdatert!